tokio-util/src/io/sync_bridge.rs

```diff
@@ -0,0 +1,46 @@
+use pin_project_lite::pin_project;
+use std::io::{Read, Write};
+use std::pin::Pin;
+use tokio::io::{AsyncRead, AsyncReadExt, AsyncWrite, AsyncWriteExt};
+
+pin_project! {
+/// Use a [`tokio::io::AsyncRead`] synchronously as a [`std::io::Read`] or
+/// a [`tokio::io::AsyncWrite`] as a [`std::io::Write`].
+#[derive(Debug)]
+pub struct SyncIOBridge<T> {
```

Comment:
also, take it or leave it, but we could probably just drop the `Io` from the name of this, since it's already in the `io` module? i'm fine either way though.

Reply:
Yeah, I was uncertain about the `Io`.  What makes me lean towards leaving it in is that there's also e.g. the whole `tokio::sync` module which is about synchronization as opposed to "synchronous".  I don't expect this type to be frequently used, and it's probably worth being a bit more verbose to highlight the conversion?   But if someone feels strongly I'm obviously fine changing!

Maybe we could use the term "blocking", e.g. `BlockingIoWrapper`?

Or actually, there's already some precendent in tokio for [from_std](https://docs.rs/tokio/1.4.0/tokio/fs/struct.File.html#method.from_std) so we could call it `StdIoWrapper` (or `StdIoBridge`)?

Reply:
I think `Blocking` or `Sync` is better than `Std` here. I don't particularly mind including `Io` in the name.

Reply:
OK...keeping this as `SyncIoBridge` then just from inertia basically.  But happy to change if someone else feels strongly.